src/advanced_inv.cpp

```diff
@@ -1901,7 +1901,7 @@ bool advanced_inv_area::is_same( const advanced_inv_area &other ) const
         id != AIM_WORN      && other.id != AIM_WORN      &&
         id != AIM_CONTAINER && other.id != AIM_CONTAINER ) {
 
-        if( veh == other.veh ) {
+        if( veh != nullptr && veh == other.veh ) {
```

Comment:
Why this? It leads to the previously fixed bug where the ALL pane shows the content of a adjacent vehicle cargo part. Example: select ALL in the left pane, select vehicle part (with items!) in the right pane. The items from the vehicle part show up in the all pane.